.github/workflows/contributors.yaml

```diff
@@ -0,0 +1,37 @@
+name: Update contributors
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:
+  push:
```

Comment:
Do we want this to just trigger on `main`?

Reply:
My thinking was that it increases visibility if it shows up e.g. in pull requests (and might be nice e.g. for someone who opens a new PR to see themselves added).

Reply:
Yeah good point. The downside is that there will be increased bot activity in Prs (readmen and this). Not sure how to weight that up

Reply:
More than one reason to only push on `main` as discussed in #447 so going with that. Done in [c6bc8c4](https://github.com/epinowcast/epinowcast/pull/446/commits/c6bc8c4b26e09478ca8da934b14d3bb82972ee51).